spec/features/courses/vacancies/edit_spec.rb

```diff
@@ -1,9 +1,11 @@
+# coding: utf-8
```

Comment:
:eyes: should this be in there?

Reply:
Ya, so this is related to [my comment in the other PR](https://github.com/DFE-Digital/manage-courses-backend/pull/723#discussion_r315004676) about replacing apostrophes with single-quotes. Emacs has detected UTF-8 characters and this is the standard way to signal that a Ruby source file is UTF-8 encoded. There is the argument that Ruby 2.x assumes UTF-8, but it doesn't hurt here and I didn't feel like hunting down the UTF-8 character.